client/client.go

```diff
@@ -368,6 +368,11 @@ func (c *Client) SetDebug(w io.Writer) {
 	c.conn.SetDebug(w)
 }
 
+// Conn returns the client's imap.Conn.
+func (c *Client) Conn() *imap.Conn {
```

Comment:
I'm not comfortable with a `Conn` method that returns an `*imap.Conn`.

Alternative ways of doing this:
* Use `client.New`, as you said in your comment
* Add `SetDeadline` function to `imap.Client`
* Add a `Timeout time.Duration` field to `imap.Client` (just like net/http)
* Add a `net.Conn` field to `imap.Client`
* Make this function return a `net.Conn`
* Another idea?

What do you think?